src/js/components/stock-movement-wizard/modals/EditPickModal.jsx

```diff
@@ -168,15 +173,22 @@ class EditPickModal extends Component {
 function validate(values) {
   const errors = {};
   errors.availableItems = [];
-
   _.forEach(values.availableItems, (item, key) => {
     if (item.quantityPicked > item.quantityAvailable) {
       errors.availableItems[key] = { quantityPicked: 'Picked quantity is higher than available' };
     }
   });
 
-  if (!values.reasonCode) {
-    errors.reasonCode = 'This field is required';
+  const pickedSum = _.reduce(
+    values.availableItems, (sum, val) =>
+      (sum + (val.quantityPicked ? _.toInteger(val.quantityPicked) : 0)),
+    0,
+  );
+
+
+  if (_.some(values.availableItems, val => !_.isNil(val.quantityPicked)) &&
+    !values.reasonCode && pickedSum !== values.quantityRequired) {
+    errors.reasonCode = 'Total quantity picked is different than requiered! Add reason code!';
```

Comment:
Small typo. Can you change "requiered" to "required".

Reply:
Sure, i missed this